modules/installation-user-infra-machines-advanced-customizing-live-iso-network-config.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso_network_keyfile_{context}"]
+= Modifying a live install image with customized network settings
+You can embed a NetworkManager keyfile into the live ISO image and pass it through to the installed system with the `--network-keyfile` flag of the `customize` subcommand.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+. Create a connection profile for a bonded interface. For example, create the `/etc/NetworkManager/system-connections/bond0.nmconnection` file with the following content:
```

Comment:
Since you are creating the `.nmconnection` file on the local system before injecting it into the ISO, I would not have the user create the file in the `/etc/NetworkManager/system-connections` location as it could adversely affect the local system.  (i.e. could change the networking config of the local system)

You can just have the user create the keyfiles (`.nmconnection` files) in the local directory.

You can then change the `--network-keyfile` options to just like so;

```
$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso \
--network-keyfile bond0.nmconnection \
--network-keyfile bond0-proxy-em1.nmconnection \
--network-keyfilebond0-proxy-em2.nmconnection
```

The `coreos-installer` command will create the files in the right location on the ISO.